frequently-encountered-problems.html

```diff
@@ -200,6 +200,39 @@ <h2>Using NTLM authentication with Node.js 17</h2>
         </p>
     </dd>
     <br />
+    <dt>
+        <h2>The relations between <code>options.trustServerCertificate, options.encrypt,</code> and <code>options.cryptoCredentialDetails</code></h2>
+    </dt>
+    <dd>
+        <ul>
+            <li>
+                If <code>encrypt = true</code> and <code>trustServerCertificate = false</code>, then <code>cryptoCredentialsDetails</code> must be provided. 
```

Comment:
I am wondering will there be some kind of error if the user does not provide "cryptoCredentialsDetails" here? if cryptoCredentialsDetails is empty, does this meaning tls.createSecureContext() will loading a empty option when try to create the secure context?